modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/src/main/java/com/liferay/headless/commerce/delivery/catalog/internal/dto/v1_0/converter/SkuDTOConverter.java

```diff
@@ -504,10 +547,61 @@ private SkuUnitOfMeasure _toSkuUnitOfMeasure(
 							cpInstanceUnitOfMeasure.getPrecision(),
 							RoundingMode.HALF_UP);
 					});
+				setTierPrices(
+					() -> {
+						if (commercePriceEntry == null) {
+							return null;
+						}
+
+						return TransformUtil.transformToArray(
+							_commerceTierPriceEntryLocalService.
+								getCommerceTierPriceEntries(
+									commercePriceEntry.
+										getCommercePriceEntryId(),
+									QueryUtil.ALL_POS, QueryUtil.ALL_POS,
+									new CommerceTierPriceEntryMinQuantityComparator(
+										true)),
+							commerceTierPriceEntry -> _toTierPrice(
+								commerceCurrency, commerceTierPriceEntry,
+								locale),
+							TierPrice.class);
+					});
 			}
 		};
 	}
 
+	private TierPrice _toTierPrice(
+		CommerceCurrency commerceCurrency,
+		CommerceTierPriceEntry commerceTierPriceEntry, Locale locale) {
+
+		BigDecimal commerceTierPriceEntryPrice =
+			commerceTierPriceEntry.getPrice();
+
+		try {
+			return new TierPrice() {
+				{
+					currency = commerceCurrency.getName(locale);
+					price = commerceTierPriceEntryPrice.doubleValue();
+					priceFormatted = StringBundler.concat(
```

Comment:
Because in this case I don't need to calculate the price with promo and discount but only have the base unit price and because I need the priceEntry for other things (get the tier price related to it).
Btw the "format" of the CommercMoney does the same "format" as here (without the currency symbol)

Reply:
reviewed together and we agreed to use priceFormatter passing the currency as parameter

Reply:
Done